.vscode/typings/meteor.d.ts

```diff
@@ -0,0 +1,2013 @@
+// Generated by typings
```

Comment:
@singhsanket143 can you remove this file from the repository?

Reply:
@peter-hank I have Removed the following file